specifyweb/frontend/js_src/lib/components/HomePage/index.tsx

```diff
@@ -49,16 +50,24 @@ function WelcomeScreenContent(): JSX.Element {
       title={welcomeText.pageTitle()}
     />
   ) : mode === 'default' ? (
-    defaultSplashScreen
+    <DefaultSplashScreen />
   ) : (
     <img alt="" className="h-full" src={source} />
   );
 }
 
-const defaultSplashScreen = (
-  <div className="relative">
-    <div className="absolute top-0 h-full w-[20%] bg-[linear-gradient(to_right,var(--background),transparent)]" />
-    <img alt="" className="w-[800px]" src={defaultWelcomePageImage} />
-    <div className="absolute top-0 right-0 h-full w-[20%] bg-[linear-gradient(to_left,var(--background),transparent)]" />
-  </div>
-);
+function DefaultSplashScreen(): JSX.Element {
+  const hueDifference = useHueDifference();
+  return (
+    <div className="relative">
+      <img
+        alt=""
+        className="w-[800px]"
+        src={defaultWelcomePageImage}
+        style={{ filter: `hue-rotate(${hueDifference}deg)` }}
+      />
+      <div className="absolute top-0 h-full w-[20%] bg-[linear-gradient(to_right,var(--background),transparent)]" />
```

Comment:
I know it was that way before, but can you please add a comment in the code above this line saying that the following 2 gradients do a "fade out" effect for the left and right edge of the SVG?

because I already forgot what this does, and it's not obvious from the code